app/models/medication.js

```diff
@@ -109,29 +102,22 @@ export default AbstractModel.extend(CanEditRequested, DateFormat, MedicationDeta
       },
       presence: {
         if(object) {
-          let isFulfilling = object.get('isFulfilling');
-          return isFulfilling;
+          return get(object, 'isFulfilling');
         }
       },
       acceptance: {
         accept: true,
         if(object) {
-          let isFulfilling = object.get('isFulfilling');
-          let requestQuantity = parseInt(object.get('quantity'));
+          let isFulfilling = get(object, 'isFulfilling');
+          let requestQuantity = parseInt(get(object, 'quantity'));
           let quantityToCompare = null;
           if (!isFulfilling) {
             // no validation needed when not fulfilling
             return false;
           } else {
             quantityToCompare = object.get('inventoryItem.quantity');
           }
-          if (requestQuantity > quantityToCompare) {
-            // force validation to fail
-            return true;
-          } else {
-            // There is enough quantity on hand.
-            return false;
-          }
+          return requestQuantity > quantityToCompare;
```

Comment:
While this technically does the same thing as before, the comments were intended to help others understand what is going on here, so I would prefer to leave them in.